src/sign/index.ts

```
@@ -0,0 +1,16 @@
+import { WrappedDocument, ProofSigningOptions, SignedWrappedDocument } from "../@types/document";
+import { sign as EcdsaSecp256k1Signature2019 } from "./EcdsaSecp256k1Signature2019";
+
+export async function sign<T = any>(
```

I'm wondering if we should throw an error if there is no target hash :thinking: